falcon/asgi/structures.py

```diff
@@ -111,7 +112,17 @@ def __init__(
 
         self.comment = comment
 
-    def serialize(self):
+    def serialize(self, handler=None):
+        """Serialize this event to string.
+
+        Args:
+            handler: Handler object that will be used to serialize the ``json`` attribute to
+                string. When not provided a default handler using the python builtin
+                json library will be used (default ``None``).
```

Comment:
Capitalize JSON here (and elsewhere).

Reply:
I meant the library name, but I'll replace with JSON